src/Network/Minio.hs

```diff
@@ -154,3 +161,11 @@ removeBucket bucket = do
 -- | Query the object store if a given bucket is present.
 bucketExists :: Bucket -> Minio Bool
 bucketExists = headBucket
+
+
+-- | Removes an ongoing multipart upload of an object.
+removeIncompleteUpload :: Bucket -> Object -> Minio ()
+removeIncompleteUpload bucket object = do
+  uploads <- listIncompleteUploads bucket (Just object) False C.$$ CC.sinkList
```

Comment:
I think this is not correct - there may be multiple on-going uploads on a single object.